kattelmod/updater.py

```diff
@@ -57,6 +57,8 @@ async def _run(self) -> None:
                     component._update_time = timestamp
```

Comment:
All the sensors are updated with the same timestamp, which leads me to believe that the order would not matter much. I was at first wondering what would happen if the same sensor got an update with an earlier timestamp, but that does not seem possible in this scheme.